docs/developer/users.mdx

```diff
@@ -130,7 +130,7 @@ mutation {
 }
 ```
 
-As a result, we get data of the newly created, but yet inactive user:
+As a result, we the get data of the newly created but yet inactive user:
```

Comment:
```suggestion
As a result, we get data of the newly created but yet inactive user:
```

Reply:
Oh, shoot. It should be "we get the data...". Nice catch, thanks.